package.json

```diff
@@ -51,8 +52,7 @@
 		"mobx-utils": "^5.5.5",
 		"react-linkify": "^1.0.0-alpha",
 		"react-markdown": "^4.0.8",
-		"uri-join": "^1.0.1",
-		"vss-web-extension-sdk": "^5.141.0"
```

Comment:
Intentional?

Reply:
Think so; I didn't manually edit this file so it must have been a transitive dep of something that changed maybe? I cal add it back and see what changes. Works without it though.